### ISSUE TITLE
Guess gesture detects the test images in ./img folder wrongly.

### ISSUE THREAD

**Issue by SumanDeedwaniya** [asingh33/CNNGestureRecognizer#70]
Hi,

I am trying to infer the gesture of test images present in ./imgs folder by loading the CNN with you pretrained weights file pretrained_weights_WinOS.hdf5 but the prediction is going wrong. Like for pppunch1.png image it is predicting PEACE gesture. Can you tell me where is it going wrong. Below is the screenshot of my output logs.
![Screenshot from 2021-07-23 17-48-48](https://user-images.githubusercontent.com/15530162/126780650-4168e7a6-286f-4bc1-92df-b4b29d27a626.png)

**Comment by asingh33** [comment]
Hey @SumanDeedwaniya 
Yeah there could be few false positives. Its not 100% accurate model. I must have copied that image file from the validation set.
